zerver/openapi/zulip.yaml

```diff
@@ -4226,6 +4226,20 @@ paths:
                                         `"role:internet"` and `"role:owners"`.
 
                                         **Changes**: New in Zulip 8.0 (feature level 209).
+                                    can_create_public_channel_group:
+                                      allOf:
+                                        - $ref: "#/components/schemas/GroupSettingValue"
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining
+                                            the set of users who have permission to create public
+                                            channels in this organization.
+
+                                            This setting cannot be set to `"role:internet"`, `"role:everyone"`,
+                                            `"role:owners"` and `"role:nobody"` system groups.
```

Comment:
This is probably worth an api design conversation about how we want to specify and describe these things; I think ideally, this information would not be in the API documentation here, but instead reference `server_supported_permission_settings` for the details on what's possible, and have the model be that a correct client should reference that data structure, and it's not an API change that we need to document if we update the values inside that object.

Reply:
Ok, will start a discussion.

Reply:
Sounds like we're converging on an answer there, but it can be a follow-up rework of this class of details in the API docs, so no reason to block this PR on it.

For reference this was https://chat.zulip.org/#narrow/stream/378-api-design/topic/disallowed.20system.20groups.20for.20group.20settings.20in.20docs/near/1816775.